guides/getting-started-tutorial/view-models.md

```diff
@@ -118,7 +118,9 @@ with this:
       cities: {
         get: function() {
           var state = this.attr('state');
-          return state && this.attr('citiesByState')[state];
+          if (state) {
+              return state & this.attr('citiesByState')[state];
```

Comment:
Looks to me like we need to remove the `if(state)` and add the missing `&` in the next line.